python/hawkey/__init__.py

```diff
@@ -201,6 +219,29 @@ def _has_just_name(self):
             not self.release and not self.arch
 
 
+class NSVCAP(_hawkey.NSVCAP):
```

Comment:
What about to rename NSVCAP() to:
ModuleID()
ModuleIdentificator()
ModuleDescriptor()
Module()
...